torch/_tensor_docs.py

```diff
@@ -1741,25 +1767,24 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('sparse_mask',
                r"""
-sparse_mask(input, mask) -> Tensor
+sparse_mask(mask) -> Tensor
 
-Returns a new SparseTensor with values from Tensor :attr:`input` filtered
-by indices of :attr:`mask` and values are ignored. :attr:`input` and :attr:`mask`
-must have the same shape.
+Returns a new :ref:`sparse tensor <sparse-docs>` with values from
+:attr:`self` tensor filtered by indices of :attr:`mask`. The values of
```

Comment:
"... from self filtered by the indices of the sparse tensor mask..."

Reply:
One thing I still don't think is clear from the one-sentence description is how the filter applies:

- is it that indices in self and mask are kept in self
- or indices in self and not in mask?

Reply:
> "... from self filtered by the indices of the sparse tensor mask..."

fixed.

Reply:
> One thing I still don't think is clear from the one-sentence description is how the filter applies:
> 
> * is it that indices in self and mask are kept in self
> * or indices in self and not in mask?

The returned sparse tensor has indices from mask. For example:
```
>>> a = torch.tensor([0, 1, 2, 3, 4])
>>> mask = torch.tensor([1, 1, 0, 1, 0]).to_sparse()
>>> a.sparse_mask(mask)
tensor(indices=tensor([[0, 1, 3]]),
       values=tensor([0, 1, 3]),
       size=(5,), nnz=3, layout=torch.sparse_coo)
```
Notice that the values of the `sparse_mask` output may contain zeros.

Updated docs include:
```
Returns a new :ref:`sparse tensor <sparse-docs>` with values from a
strided tensor :attr:`self` filtered by the indices of the sparse
tensor :attr:`mask`. The values of :attr:`mask` sparse tensor are
ignored. :attr:`self` and :attr:`mask` tensors must have the same
shape.

.. note::

  The returned sparse tensor has the same indices as the sparse tensor
  :attr:`mask`, even when the corresponding values in :attr:`self` are
  zeros.
```